README.md

```diff
@@ -43,12 +44,13 @@ If a file is executable, it is directly invoked, otherwise it is interpreted wit
 Hooks related to `commit` events will also have a `${STAGED_FILES}` environment variable set, that is the list of staged and changed files (according to `git diff --cached --diff-filter=ACMR --name-only`), one per line and where it makes sense (not `post-commit`). If you want to iterate over them, and expect spaces in paths, you might want to set `IFS` like this.
 
 ```shell
-IFS="
-"
-
+IFS="$IFS_NEWLINE"
```

Comment:
We don't say what is `IFS_NEWLINE` here.
But also, I'm not too sure if this extra complexity is needed in this code example?
Unless something resets `IFS` inside the loop, you don't need this.

Reply:
ah jeah, sure, this is code in the hook which we execute in a new shell...